docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<p>"...is necessary because each time <code>step</code> is called...": is a word missing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I wasn't quite sure where the issue was but I rewrote that section to be clearer in either case.

Reply:
<p>Much clearer, thanks!</p><p> </p>